core/garment/src/garment.ts

```diff
@@ -753,13 +753,14 @@ async function garmentFromWorkspace(
         const outputPaths = output ? [...arrayfy(output)] : [];
         if (
           currentActionGraph.getDirectDependantsOf(action).length &&
-          !outputPaths.length
+          !outputPaths.length &&
+          filesToWrite.length
```

Comment:
@beshanoe I believe that this is the true origination of the defect.  I tested in the reported repo and it works but please verify though that I'm not breaking anything as I'm still not too familiar with the surrounding code.

What was happening was that for "next" actions, `actionToOutputPath(action)` was added to the meta.outputPath, but because the runner had no output, `filesToWrite` was empty and so the loop below which creates the directories if they don't exist, would never be entered.  When the next action ran, if it had no input, it looked at `meta.outputPath` and expected it to exist, but it does not. 

![image (24)](https://user-images.githubusercontent.com/34415120/95782639-1ded1000-0cc8-11eb-8c50-5293f6d860e1.png)

Reply:
I've check and I think your solution is good enough👍

Reply:
Nice, thanks 🙏 !